basis/io/directories/unix/unix.factor

```diff
@@ -67,7 +67,8 @@ M: unix copy-file ( from to -- )
 
 : >directory-entry ( dirent* -- directory-entry )
     [ d_name>> alien>native-string ]
-    [ d_type>> dirent-type>file-type ] bi
+    [ drop f ] ! TODO: set size slot
```

Comment:
Maybe add to windows and we'll make a TODO to generalize if makes sense.  I think on linux/mac you need to make another system call (``stat``) to get the file sizes while iterating, which might be worth doing always, but might not...